### ISSUE TITLE
Can't use JSON keys with periods

### ISSUE THREAD

**Issue by robwierzbowski** [Updater/node-sass-json-importer#12]
My JSON file has keys with periods in them: `fontname.extension`. When I import using NSJI I get:

```
error in plugin 'sass'
Message:
    ../../.tmp/manifests/rev-manifest.json
Error: expected ':' after $charter-bt-italic in assignment statement
        on line 1 of ../../.tmp/manifests/rev-manifest.json
>> $charter-bt-italic.woff: charter-bt-italic-fc2cd13f73.woff;
   ^
```

I read up a bit on the other issues with string quoting, and realize it's not so easy. Do you think this is solvable?

**Comment by pmowrer** [comment]
Is `$charter-bt-italic.woff`[a valid SASS variable name](http://stackoverflow.com/a/17194994/89594)? Maybe, but I think the period needs to be escaped. Try naming the key `$charter-bt-italic\.woff`.

**Comment by pmowrer** [comment]
@robwierzbowski Any luck? Closing until further updates.

**Comment by robwierzbowski** [comment]
I abandoned this route and went with a custom node sass function. Thanks
for closing.

On Wednesday, December 9, 2015, Patrick Mowrer <notifications@github.com>
wrote:

> Closed #12 <https://github.com/Updater/node-sass-json-importer/issues/12>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Updater/node-sass-json-importer/issues/12#event-487096057>
> .
>


-- 
Rob Wierzbowski
@robwierzbowski <http://twitter.com/#!/robwierzbowski>
http://github.com/robwierzbowski
http://robwierzbowski.com

**Comment by brod-ie** [comment]
What's an up-to-date workaround for this? Ideally there'd be an option on the `jsonImporter` function to skip keys containing such periods?

For example, I want to make use of this library with the [Design Tokens plugin for Figma](https://www.figma.com/community/plugin/888356646278934516). Their outputted JSON contain an `extensions` object which always contains subkeys containing periods:
```
...
"extensions": {
    "org.lukasoppermann.figmaDesignTokens": {
      ...
  }
}
...
```
Resulting in the following error from NSJI:
```
Error: Invalid CSS after "...xtensions: (org": expected expression (e.g. 1px, bold), was ".lukasoppermann.fig"
```
